messaging-experiments-82.json

```diff
@@ -1058,5 +1058,888 @@
         }
       }
     ]
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-SET-DEFAULT-V1",
+    "enabled": true,
+    "targeting": "firefoxVersion >= '82.' && browserSettings.update.channel != 'release' && region == 'US' && localeLanguageCode == \"en\" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'bug-1675623-message-multi-stage-onboarding-set-default-experiment-release-82-85' in activeExperiments) && 'app.shield.optoutstudies.enabled'|preferenceValue",
```

Comment:
realised on testing localeLanguageCode == \"en\" is failing targeting and should be updated to localeLanguageCode == 'en', will update  patch